nix/local/envs.nix

```diff
@@ -6,6 +6,8 @@ let
     alejandra
     shfmt
     yq
+    git
+    git-subrepo
```

Comment:
Can we leave this impure?

Reply:
I guess. Git probably isn't necessary. It was more for the git-subrepo, which most people won't have installed.